mantle/kola/tests/ignition/kdump.go

```diff
@@ -0,0 +1,205 @@
+package ignition
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/coreos/coreos-assembler/mantle/kola"
+	"github.com/coreos/coreos-assembler/mantle/kola/cluster"
+	"github.com/coreos/coreos-assembler/mantle/kola/register"
+	"github.com/coreos/coreos-assembler/mantle/platform"
+	"github.com/coreos/coreos-assembler/mantle/platform/conf"
+	"github.com/coreos/coreos-assembler/mantle/platform/machine/qemu"
+	"github.com/coreos/coreos-assembler/mantle/util"
+)
+
+// Test kdump to remote hosts
+
+func init() {
+	// Create 0 cluster size to allow starting and setup ssh server as needed for the test
+	// See: https://github.com/coreos/coreos-assembler/pull/1310#discussion_r401908836
+	register.RegisterTest(&register.Test{
+		Run:         kdumpSSHTest,
+		ClusterSize: 0,
+		Name:        `kdump.crash.ssh`,
+		Description: "Verifies kdump logs are exported to SSH destination",
+		Flags:       []register.Flag{},
+		Distros:     []string{"rhcos", "fcos"},
+		Tags:        []string{"kdump", kola.NeedsInternetTag, "reprovision"},
+	})
+}
+
+// TangServer contains fields required to set up a tang server
+type SshServer struct {
+	Machine           platform.Machine
+	MachineAddress    string
+	SSHPort           string
+	PrivSSH           string
+	PubSSH            string
+}
+
+
+// this simply start a VM and retrieve the SSH thumprint for it
```

Comment:
```suggestion
// Start a VM and return the SSH key pair
```

Reply:
Thanks for spotting the leftover :) I was originally extracting the thumbprint with `ssh-keyscan` but ended up turning off `StrictHostChecking` on the other machine instead